ibm/mas_devops/roles/ocp_setup_mas_deps/defaults/main.yml

```diff
@@ -1,25 +0,0 @@
----
```

Comment:
Why was this file deleted?

Reply:
the file was deleted by mistake. add it back.